src/xenia/apu/xma_context.h

```diff
@@ -168,28 +167,30 @@ class XmaContext {
   void set_is_enabled(bool is_enabled) { is_enabled_ = is_enabled; }
 
  private:
+  static void SwapInputBuffer(XMA_CONTEXT_DATA * data);
```

Comment:
In Xenia code style, the * is not separated from the type — please run clang-format (`xb format` between `git add` and `git commit` after reverting, or `clang-format -i` explicitly for a whole file if it's not in Git staging).

Reply:
I've noticed Visual Studio can get hung up on this sometimes depending on the surrounding code and ends up formatting pointers this way. Not sure if `xb format`/`xb lint` also gets hung up with this.